tests/unit/repo/plots/test_data.py

```diff
@@ -39,10 +23,22 @@ def test_finding_lists(dictionary, expected_result):
     assert list(result) == expected_result
 
 
-@pytest.mark.parametrize("fields", [{"x"}, set()])
-def test_finding_data(fields):
-    data = {"a": {"b": [{"x": 2, "y": 3}, {"x": 1, "y": 5}]}}
```

Comment:
Expanded scope of the test to test `to_datapoints` and remove redundancy in tests (`test_finding_data` was checking basically the same as `test_find_data_in_dict`).